components/glew/Makefile

```diff
@@ -0,0 +1,100 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Jim Klimov <jimklimov@gmail.com>
+#
+
+# NOTE: This project links with /usr/openwin/lib/libGL.so.1, which unrolls
```

Comment:
This is two-fold:
1) Linking produces a bad binary (references paths that may not exist on cutormer's deployment) if VBoxOGL is the target of libGL.so.1 symlinks
2) Packaging complains about library not in this repo... or something of the sort. It complains without the pkk.depend... flag, it builds with it ;)